### ISSUE TITLE
Call set_gating_sequences() in file  one_publisher_to_one_unicast_throughput_test.cc

### ISSUE THREAD

**Issue by paulran** [fsaintjacques/disruptor--#33]
[master/perf/one_publisher_to_one_unicast_throughput_test.cc#L70](https://github.com/fsaintjacques/disruptor--/blob/master/perf/one_publisher_to_one_unicast_throughput_test.cc#L70)

Why do not call ring_buffer.set_gating_sequences()  before this line, although the file just is for performance testing?

This is unfavorable for beginners, and is not precise.

**Comment by tvdijk** [comment]
I agree. Also, in one_publisher_to_three_pipeline_throughput_test, the publisher there will also not wait for any consumer, it should wait for third_processor.

**Comment by fsaintjacques** [comment]
Please open a PR, I don't have much personal time to work on this project.
